src/constants/icons.js

```diff
@@ -0,0 +1,9 @@
+const eye = require('../assets/icons/eye.png');
+const check = require('../assets/icons/check.png');
+const google = require('../assets/icons/google.png');
+
+export default {
+    eye,
+    check,
```

Comment:
The same about indentation and ESLint, comments about that that I left in the MainStackNavigator.js file.